vizro-ai/mkdocs.yml

```diff
@@ -9,14 +9,16 @@ nav:
       - Run methods: pages/user-guides/run-vizro-ai.md
       - Create advanced charts: pages/user-guides/create-advanced-charts.md
       - Use different languages: pages/user-guides/use-different-languages.md
-  - Explanation:
+  - Find out more:
       - FAQs: pages/explanation/faq.md
       - Disclaimer: pages/explanation/disclaimer.md
       - Safeguard code execution: pages/explanation/safeguard.md
       - Safety in Vizro-AI: pages/explanation/safety-in-vizro-ai.md
-  - Contribute:
-      #      - Contributing: pages/contribute/contributing.md
       - Authors: pages/contribute/authors.md
+  #- Contribute:
```

Comment:
Could we have a card that just links to the contribution guidelines for Vizro-AI, or similar. We probably need to double check that the instructions are correct for both backages (there might be very slight differences)

Reply:
There are no contribution guidelines right now, which is why I removed it.

Reply:
Apologies, I meant `Vizro` of course. Since they are both part of the same mono-repo, the guidelines should be the same. Apart from very slight differences (e.g. `Run hatch -v env create from the vizro-core`) all is the same. Maybe we could add tabs for the small information differences ([like so](https://squidfunk.github.io/mkdocs-material/reference/content-tabs/#linked-content-tabs-feature-enabled)) and then link to the same page

Reply:
I've made a new ticket for this work: https://github.com/McK-Internal/vizro-internal/issues/708